Makefile

```diff
@@ -1,6 +1,6 @@
 ASSISTED_SWARM = build/assisted-swarm
 CONTAINER_COMMAND := $(or $(CONTAINER_COMMAND),docker)
-IMAGE := $(or $(IMAGE),quay.io/openshift/assisted-swarm:latest)
+IMAGE := $(or $(IMAGE),quay.io/oamizur/assisted-swarm:latest)
```

Comment:
Did you commit this intentionally?

Reply:
For now, until we will have an image available in openshift

Reply:
Alright. Should probably be edge-infrastructure though